app/main.py

```diff
@@ -1,8 +1,55 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
```

Comment:
Is it similar to what I have shown you in the previous comment?

Reply:
Sorry. Missed the tail =)